README.md

```diff
@@ -34,6 +34,11 @@ We are still in early development of the multi-backend rewrite and things may ch
 
 [Contributing](docs/contributing.md).
 
+### Dependencies
+
+Two dependencies need to be installed system wide. The [LunarG Vulkan SDK](https://vulkan.lunarg.com/sdk/home) and [mono](https://www.mono-project.com/). Instructions for installation are on their websites.
```

Comment:
I don't really know if mono should be a system dependency. I'm not sure how hard it would be to build ourselves so this is fine for now

Reply:
We probably don't want to build it on our own, this would take a very long time...

Reply:
It's fine on linux, but on windows it's a compound msvc (for c++) + cygwin (for c#) build which sounds extraordinarily annoying.

Reply:
Alright. In that case let's just say the specific version of mono that we're using

Reply:
Done.